connectivity/FEATURE_BLE/CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# cordio libraries are part of ble stack (_ll is optional)
+# We use INTERFACE libraries to form the main object library otherwise we would 
+# get circular deps error between ble <-> cordio
+add_library(mbed-ble-cordio INTERFACE)
+add_library(mbed-ble-cordio-ll INTERFACE)
```

Comment:
Is there a reason `mbed-ble-cordio-ll` is defined here? Should it be in `connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/CMakeLists.txt`.

Reply:
I moved them. I wanted to have them in one place with just one comment and let the deps fixed and remove this comment. I spread it now